src/applications/pre-need/config/form.jsx

```diff
@@ -368,17 +368,12 @@ const formConfig = {
                       'ui:title': 'File name',
                     },
                   })
-                : fileUploadUI('Select files to upload', {
-                    buttonText: 'Upload file',
-                    addAnotherLabel: 'Upload another file',
+                : fileUploadUi({
                     fileUploadUrl: environment.isProduction()
                       ? `${environment.API_URL}/v0/preneeds/preneed_attachments`
                       : `${
                           environment.API_URL
                         }/simple_forms_api/v1/simple_forms/submit_supporting_documents`,
-                    fileTypes: ['pdf'],
-                    maxSize: 15728640,
-                    hideLabelText: true,
                     createPayload: !environment.isProduction()
```

Comment:
This is missing from the integration version. Is there a reason you took it out there?

Reply:
Yes, those properties are now being handled in the new `upload,js` component.